controllers/users.js

```diff
@@ -6,11 +6,18 @@ SALT_LENGTH = 12
 
 router.get("/:userId", async (req, res) => {
 	try {
-		const userProfile = await User.findById(req.params.userId)
+		const userProfile = await User.findById(req.params.userId).populate(
+			"ownedReviews"
+		)
 		if (!userProfile) {
 			res.status(404)
 			throw new Ettor("User not found")
```

Comment:
```suggestion
			throw new Error("User not found")
```

Reply:
This method of throwing an error is slightly different to what we've applied elsewere but we can spend some time making things more consistent later. For now I'm happy to approve this. Just need to correct this typo.